_studio/enctools/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+set(MFX_ORIG_LDFLAGS "${MFX_LDFLAGS}" )
+
+mfx_include_dirs( )
+
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/include )
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/aenc/include )
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/shared/include)
+
+set(sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/enctools.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/mfx_enctools_brc.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/mfx_enctools_common.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/mfx_enctools_aenc.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/mfx_enctools_utils.cpp
+    )
+set( sources.plus "" )
+
+set( defs "${API_FLAGS} ${WARNING_FLAGS}" )
+
+if (MFX_ENABLE_AENC)
+  find_library( AENC_LIBRARY libaenc.a PATHS ${CMAKE_HOME_DIRECTORY}/contrib/aenc/ )
```

Comment:
I don't quite get why you need so many enable defines:
* MFX_ENABLE_AENC
* MFX_ENABLE_ENCTOOLS
* MFX_ENCTOOLS_ON

It seems that at least one of them is not needed. Can we have the following?
* MFX_ENABLE_ENCTOOLS=ON|OFF, OFF by default
* MFX_ENABLE_AENC=ON|OFF, OFF by default